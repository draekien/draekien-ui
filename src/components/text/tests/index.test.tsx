/** @jsxRuntime classic */
/** @jsx jsx */
import { render } from '@testing-library/react';
import { jsx } from 'theme-ui';
import Text from '..';

describe('Text component', () => {
  test('should match snapshot with default props', () => {
    const { container } = render(<Text>test</Text>);

    expect(container).toMatchSnapshot();
  });

  test('should match snapshot with custom props', () => {
    const { container } = render(
      <Text
        color="primary"
        fullWidth={true}
        align="center"
        variant="hero"
        truncate={true}
        margin="0px"
      >
        Test
      </Text>
    );

    expect(container).toMatchSnapshot();
  });
});
