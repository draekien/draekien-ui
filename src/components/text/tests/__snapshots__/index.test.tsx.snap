// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component should match snapshot with custom props 1`] = `
<div>
  <div
    class="css-1b29sq2-Text"
  >
    Test
  </div>
</div>
`;

exports[`Text component should match snapshot with default props 1`] = `
<div>
  <div
    class="css-3vy6se-Text"
  >
    test
  </div>
</div>
`;
